changelog/README.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 21st February 2025
+* Updated booking [relationship](../operations/bookings.md#Response) with orders (multiple orders now related to booking instead of one).
```

Comment:
In `booking_relationships` we should logically be pointing to a new data type for `orders`, but I see the definition hasn't changed because the type is not stated, it just says "object". You can't have a type that's just "object", without saying what the object is. That's true for `tables`, `customer` and `orders`.